ingress-examples/README.md

```diff
@@ -14,5 +14,20 @@
 
 [ingress-subdir-non-root.yaml](ingress-subdir-non-root.yaml): use your.com/kubecost instead of kubecost on the root path
 
+## Quickstart if using Nginx Ingress Controller (`ingressClassName: nginx`)
 
```

Comment:
I recommend to add a few lines here to elaborate more about the Nginx ingress controller and user can find more details at Nginx doc https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/

Reply:
@linhlam-kc Will do!